nss-evening-cohort-7/nutshell-flying-buttresses
# installed momentJS

Installed momentJS
To Test: 
1. fetch branch and checkout
2. cd into lib
3. npm install
4. uncomment console.log (line 6 in main.js)
5. hs -o to open in browser
6. Look at console- you should see the current time
